oozie-to-airflow/mappers/email_mapper.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import subprocess
+
+import jinja2
+from airflow.utils.trigger_rule import TriggerRule
+
+from definintions import ROOT_DIR
+from mappers.action_mapper import ActionMapper
+from utils import el_utils, xml_utils
+
+
+class EmailMapper(ActionMapper):
+    """
+    Converts an email oozie node to Airflow operator.
+
+    In order to use this, the user must specify an Airflow connection to use, and
+    provide the password there.
+
+    An action node looks like this, but we get passed the <email> node.
+
+    <action name="[NODE-NAME]">
+        <email xmlns="uri:oozie:email-action:0.2">
+            <to>[COMMA-SEPARATED-TO-ADDRESSES]</to>
+            <cc>[COMMA-SEPARATED-CC-ADDRESSES]</cc> <!-- cc is optional -->
+            <bcc>[COMMA-SEPARATED-BCC-ADDRESSES]</bcc> <!-- bcc is optional -->
+            <subject>[SUBJECT]</subject>
+            <body>[BODY]</body>
+            <content_type>[CONTENT-TYPE]</content_type> <!-- content_type is optional -->
+            <attachment>[COMMA-SEPARATED-HDFS-FILE-PATHS]</attachment> <!-- attachment is optional -->
+        </email>
+        <ok to="[NODE-NAME]"/>
+        <error to="[NODE-NAME]"/>
+    </action>
+    """
+
+    def __init__(self, oozie_node, task_id,
```

Comment:
What do we do if there are fields in the node that we don't parse/extract? Can we log it somehow, so that the user is aware of this?